### PR TITLE
better error message for javascript parse errors

### PR DESCRIPTION
**Pull request by drewfish**

### CODE REVIEW

**Comment by caridy** [comment]
+1

**Comment by isao** [comment]
nice

File: lib/app/addons/rs/yui.server.js [hunk 1]
```diff
@@ -626,7 +626,7 @@ YUI.add('addon-rs-yui', function(Y, NAME) {
                 libvm.runInNewContext(file, ctx, res.source.fs.fullPath);
             } catch (e) {
                 yui = null;
-                Y.log(e.message + '\n' + e.stack, 'error', NAME);
+                Y.log('failed to parse javascript file ' + res.source.fs.fullPath, 'error', NAME);
```
Comment (FabianFrank) [inline lib/app/addons/rs/yui.server.js:629]: @drewfish doesn't the stack contain the parsing error and line number?

Comment (drewfish) [inline lib/app/addons/rs/yui.server.js:629]: The stack doesn't.  I think the e.message contains the error, so yeah that should be in the output as well.

**Pull request merged**